Sources/HiPMobile/BusinessLayer/ContentApiFetchers/BaseDataFetcher.cs

```diff
@@ -87,12 +86,16 @@ public BaseDataFetcher(IRoutesBaseDataFetcher routesBaseDataFetcher, IExhibitsBa
                     transaction.Rollback();
                     return;
                 }
-                dataToRemoveFetcher.CleaupRemovedData();
                 if (token.IsCancellationRequested)
                 {
                     transaction.Rollback();
                 }
             }
+            await dataToRemoveFetcher.FetchDataToDelete (token);
```

Comment:
There space in front of the parentheses should be removed on this line, line 95 and 97.